composer.json

```diff
@@ -19,8 +19,8 @@
   },
   "require":{
     "php":">=5.3.2",
-    "contao/core":">=3.0,<4",
-    "contao-community-alliance/composer-plugin":"*",
+    "contao/core-bundle": ">=3.0 || ~4.1",
```

Comment:
I would suggest: ""contao/core-bundle":"~3.5 || ~4.4","

3.5 is last 3.x LTS version
4.4 is the current 4.x version